starlark/capture_local_test.go

```diff
@@ -70,6 +70,7 @@ func TestCaptureLocalFunc(t *testing.T) {
 					{starlark.String("workdir"), starlark.String("/tmp/capturecrashd")},
 					{starlark.String("file_name"), starlark.String("echo.txt")},
 					{starlark.String("desc"), starlark.String("echo command")},
+					{starlark.String("append"), starlark.True},
```

Comment:
Please add a new test case that actually validates whether content is appended to the file.   
You can create file at this location first `filepath.Join("/tmp/capturecrashd", "echo.txt")` then run `captureLocalFunc(newTestThreadLocal(t), nil, nil, kwargs)` then validate the content.